src/oneD/StFlow.cpp

```diff
@@ -780,6 +804,8 @@ AnyMap StFlow::getMeta() const
 
     state["Soret-enabled"] = m_do_soret;
 
+    state["Flux-Gradient-Basis"] = static_cast<long int>(m_fluxGradientBasis);
```

Comment:
Our naming convention for YAML fields is lower case except for proper names, so this should be `flux-gradient-basis`.